deploy/UST.anchorStrategy.ts

```diff
@@ -11,33 +11,52 @@ const func: DeployFunction = async function (env: HardhatRuntimeEnvironment) {
   const aust = await get("aUST");
   const vault = await get("Vault_UST");
 
-  const { multisig, ethAnchorRouter, perfFeePct } =
-    await getCurrentNetworkConfig();
+  const { multisig, ethAnchorRouter } = await getCurrentNetworkConfig();
 
   const treasury = env.network.config.chainId == 1 ? multisig : deployer;
   const owner = env.network.config.chainId == 1 ? multisig : deployer;
 
-  await deploy("Vault_UST_AnchorStrategy", {
-    contract: "AnchorUSTStrategy",
+  const MockChainlinkPriceFeedFactory = await deploy("MockChainlinkPriceFeed", {
```

Comment:
this doesn't look right. this file is suposed to be for mainnet deployments as well. why are we deploying a mock?

My guess is this should:
* be it's own deployment file
* deployment called `ChainlinkPriceFeed` instead of `MockChainlinkPriceFeed`
* we manually fill `deployments/mainnet` with the actual price feed we'll use there. that will the separate deploy script will be skipped on mainnet

Reply:
I don't think I would, no :grimacing: 
keep a piece of test-only code in a real deployment file would look really weird when coming back to this in the near future

but to be honest, I don't think this is complicated at all. pretty much just move this deploy call to a separate file, and add that file as a dependency here. or does it mess with anything else that I'm forgetting?

Reply:
ok I went ahead and implemented this, deployed a new set of ropsten contracts to be sure it still works so the JSONs got updated with the last commit too, should be OK though?